sites/footers/old-footers-en.html

```diff
@@ -7,17 +7,17 @@
 ],
 "altLangPage": "old-footers-fr.html",
 "secondlevel": false,
-"dateModified": "2023-12-04",
+"dateModified": "2024-11-12",
 "share": "true"
 }
 ---
 <div class="wb-prettify all-pre hide"></div>
 
-<h2>Version 6.0</h2>
+<h2>Version 7.0</h2>
```

Comment:
So we're jumping straight from version 5.0 to version 7.0 now? Please keep version 6.0 and add the version 7.0 above it. **Please make sure the code sample matches each version**. Do the same for French.